tests/foreman/longrun/test_inc_updates.py

```diff
@@ -286,9 +286,10 @@ def test_positive_incremental_update_time(module_target_sat, module_sca_manifest
     # expect: incr. "version-1.1" is created
     update_start_time = datetime.utcnow()
     result = module_target_sat.cli.ContentView.version_incremental_update(
-        {'content-view-version-id': cvv['id'], 'errata-ids': REAL_RHEL8_1_ERRATA_ID}
+        options={'content-view-version-id': cvv['id'], 'errata-ids': REAL_RHEL8_1_ERRATA_ID},
+        format='base',
     )
-    assert 'version-1.1' in str(result[0].keys())
+    assert f'Content View: {cv.name} version 1.1' in result
```

Comment:
I wonder why changing the output format to base and assert it this new way?

Reply:
`hammer content-view incremental-update` with output format `base` gives/prints meaning data like below, while all other format doesn't. Validated this in dugging mode
```
Content View: VyFPexkvDT version 1.2\nAdded Content:
```